### PR TITLE
94 show own citations

### LINKED ISSUE #94: Bank — view your citations
Based on this screen:

Screen Shot 2023-08-01 at 1 17 11 PM

### PR DESCRIPTION
**Pull request by hangler**
Fix #94.

### CODE REVIEW

File: app/routes/bank.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export default function BankIndex() {
+  return (
+    <div className="p-12">
+      <PageHeader>Bank of Canadian English</PageHeader>
+      <nav className="bg-slate-100">
+        <h2 className="my-8 text-xl font-bold">
+          Links: <NavLink to="/bank/stats">Statistics</NavLink> •{" "}
+          <NavLink to="/bank/create">New</NavLink> •{" "}
+          <NavLink to="/bank/own">Show own</NavLink>
```
Comment (Shawk031) [inline app/routes/bank.tsx:20]: Right now this is throwing an error for me-- I think it's cause I'm not in my local database of users, but the error is uncaught so I don't know if we want to change that or not.

I'll add my user to my local database and see how that changes things

Comment (Shawk031) [reply app/routes/bank.tsx:20]: update: it does fix it

**Review by Shawk031 — APPROVED** [review]
looks good! just that one thing

File: app/routes/bank.tsx (hunk 1 above)
Comment (hangler) [reply app/routes/bank.tsx:20]: Good catch. I'll create a separate issue for this.

**Pull request merged**